buildengine/build_go_test.go

```diff
@@ -34,26 +34,30 @@ func TestGenerateGoModule(t *testing.T) {
 				&schema.Data{Name: "EchoRequest"},
 				&schema.Data{Name: "EchoResponse"},
 				&schema.Verb{
-					Name:     "echo",
-					Request:  &schema.Ref{Name: "EchoRequest"},
-					Response: &schema.Ref{Name: "EchoResponse"},
+					Name:       "echo",
+					Visibility: "internal",
```

Comment:
Use the constants here and elsewhere rather than literals - it will result in a compiler error if they get renamed, unlike the literals which will silently add invalid data.